ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -20,12 +20,22 @@ images:
       as:
       - quay.io/app-sre/boilerplate:image-v2.1.0
   to: oadp-operator
+- dockerfile_path: build/ci-Dockerfile
+  from: ocp_builder_rhel-8-golang-1.16-openshift-4.9
+  to: test-oadp-operator
 operator:
   bundles:
   - dockerfile_path: build/Dockerfile.bundle
   substitutions:
   - pullspec: quay.io/konveyor/oadp-operator:latest
-    with: oadp-operator
+    with: test-oadp-operator
+promotion:
+  additional_images:
+    test: test-oadp-operator
```

Comment:
This entry promotes the `test-oadp-operator` to a central registry when we promote - after merges happen to the repo.

Reply:
so, pipeline: test-oadp-operator to openshift/test-oadp-operator:test ?